Assets/Scripts/Tests/Documents/Neues RTF-Dokument.rtf

```diff
@@ -0,0 +1,244 @@
+{\rtf1\adeflang1025\ansi\ansicpg1252\uc1\adeff31507\deff0\stshfdbch31505\stshfloch31506\stshfhich31506\stshfbi31507\deflang1031\deflangfe1031\themelang1031\themelangfe0\themelangcs0{\fonttbl{\f0\fbidi \froman\fcharset0\fprq2{\*\panose 02020603050405020304}Times New Roman;}{\f2\fbidi \fmodern\fcharset0\fprq1{\*\panose 02070309020205020404}Courier New;}
```

Comment:
Please give this document a better name. 
Also, please use a different format. RTF is not very human-readable. I'd recommend using a `.md` file